samza-kv-rocksdb/src/main/scala/org/apache/samza/storage/kv/RocksDbKeyValueStore.scala

```diff
@@ -91,7 +91,13 @@ object RocksDbKeyValueStore extends Logging {
         .toSet
 
       (configuredMetrics ++ rocksDbMetrics)
-        .foreach(property => metrics.newGauge(property, () => rocksDb.getProperty(property)))
+        .foreach(property => metrics.newGauge(property, () =>
+          if (rocksDb.isOwningHandle) {
```

Comment:
Minor: Maybe add a comment explaining that isOwningHandle is false if the db is closed.

Reply:
Makes sense. Added the comment.